### PR TITLE
add 2023 OSS NA presentation by sagikazarmark

### PR DESCRIPTION
**Pull request by sagikazarmark**

### CODE REVIEW

File: presentations.yaml [hunk 1]
```diff
@@ -113,4 +113,28 @@
...
+  presenters:
+    - name: Márk Sági-Kazár
+      github: sagikazarmark
+  event:
+    name: OSS NA 2023
+    url: https://ossna2023.sched.com/event/1K5FED
+  language: EN
+  projects:
+    - Kubernetes
+    - External Secrets
```
Comment (csantanapr) [inline presentations.yaml:138]: External Secrets is not a cncf project, 
@castrojo @cjyabraham What do you think? Maybe we should update the field to `cncfProjects:` ?

Comment (csantanapr) [reply presentations.yaml:138]: @sagikazarmark please remove `External Secrets` as we update the policy and a PR template to make this more clear

Comment (cjyabraham) [reply presentations.yaml:138]: @csantanapr yes, good idea!

Comment (sagikazarmark) [reply presentations.yaml:138]: @csantanapr it's a Sandbox project: https://www.cncf.io/projects/external-secrets/

Comment (castrojo) [reply presentations.yaml:138]: I'll add a note in the readme that sandbox is ok.

**Review by castrojo — APPROVED** [review]

**Pull request merged**